### PR TITLE
#163891995 Implement LFA feedback

### PR DESCRIPTION
**Pull request by mogoria**
#### What does this PR do?
implements feedback from the LFA

#### Description of Task to be completed?

- [x] Ensure that party names are unique for every request to create a party.
- [x] Edit endpoint should display an appropriate message and status code
- [x] GET requests for a single party, there is an issue when we have a payload
- [x] User should not create an office with the same name twice and return an appropriate error message
- [x] Return appropriate validation message for updating a party
- [x] Remove id from payload when creating parties and offices, should be auto-generated
- [x] Validate party types to ensure that a user supplies only a party particular type that matches either federal, legislative, state or local government

#### How should this be manually tested?

1. Clone this repository
2. Consult readme for installation instructions
3. Run the application
4. Using postman, make desired requests to the endpoints provided in the README
 
#### What are the relevant pivotal tracker stories?
[#163891995](https://www.pivotaltracker.com/n/projects/2241724/stories/163891995)

#### Screenshots
*Removed id from payload*
image

*Cannot create party with existing name*
image

*Edit party displays appropriate message and response*
image

*Create party validates the type*
image

### CODE REVIEW

**Comment by artorious** [comment]
Great implementation of feedback. Have you looked into what is causing the build to fail?

File: .coverage [hunk 1]
```diff
@@ -1 +1 @@
...
...\\app\\api\\v1\\views\\political_party_route.py":[1,2,3,4,5,6,8,10,45,55,64,88,13,14,16,18,19,20,28,29,30,31,35,36,37,38,39,91,92,93,67,68,70,71,73,74,77,78,79,80,81,82,48,50,51,52,58,59,60,61,33,41,42,43,23,24,25,94,85,86,75,53,62],"F:\\Github\\Politico\\app\\api\\v1\\models\\__init__.py":[1],"F:\\Github\\Politico\\app\\api\\v1\\models\\political_party_model.py":[1,2,3,5,19,32,36,44,51,59,46,47,49,7,9,11,60,61,12,13,14,16,17,53,38,39,40,41,54,55,56,57,21,22,23,24,26,27,62,34,48,42,58],"F:\\Github\\Politico\\app\\api\\v1\\views\\utils.py":[1,2,3,4,16,22,29,35,42,43,45,48,53,58,64,72,79,85,91,92,97,116,117,123,30,31,32,33,36,37,39,40,93,46,94,95,99,100,103,104,106,80,81,54,55,56,83,108,86,87,65,73,74,75,76,67,68,70,89,110,49,50,112,18,19,6,8,9,12,13,51,111,23,24,25,27,26,77,66,118,119,120,121,125,126,127,130,131,132,135,82,136,137,88,59,60,138,139],"F:\\Github\\Politico\\app\\api\\v1\\views\\political_office_route.py":[1,2,3,4,5,6,8,9,39,53,14,15,16,18,24,25,26,30,31,32,33,42,43,45,46,47,56,57,58,59,34,35,36,49,50,60,27,28,19,20,21],"F:\\Github\\Politico\\app\\api\\v1\\models\\political_office_model.py":[1,3,6,8,23,27,34,41,11,36,37,39,16,42,44,17,18,20,21,43,25,29,30,31,38,13,32]}}
```
Comment (artorious) [inline .coverage]: Add .coverage to your `.gitignore`

**Comment by mogoria** [comment]
> 
> 
> Great implementation of feedback. Have you looked into what is causing the build to fail?

I had a drop in quality from codeclimate and codacy. I'll try to work on the issues.

File: .coverage (hunk 1 above)
Comment (mogoria) [reply .coverage]: great note

**Pull request merged**